policyengine_us/parameters/gov/states/mo/tax/income/rates.yaml

```diff
@@ -1,49 +1,68 @@
 description: The tax rates for individual residents in Missouri. Thresholds are <=.
 brackets:
   - rate:
-      2019-01-01: 0.015  
+      2019-01-01: 0.015
     threshold:
-      2019-01-01: 0
+      2019-01-01: 104
```

Comment:
This isn't correct: The $104 is just for rounding purposes, the bracket starts at $0